src/main/resources/assets/spookytime/blockstates/witched_pumpkin.json

```diff
@@ -1,20 +1,5 @@
 {
 	"variants": {
-		"facing=east": {
-			"model": "spookytime:block/witched_pumpkin",
-			"y": 90
-		},
-		"facing=south": {
-			"model": "spookytime:block/witched_pumpkin",
-			"y": 180
-		},
-		"facing=west": {
-			"model": "spookytime:block/witched_pumpkin",
-			"y": 270
-		},
-		"facing=north": {
-			"model": "spookytime:block/witched_pumpkin",
-			"y": 0
-		}
+		"": {"model": "spookytime:block/witched_pumpkin"}
```

Comment:
did you remove the BlockState in the class?

Reply:
What? You need to explain what you mean a bit more. I did rename the old witched pumpkin (tiny one) to witched tiny pumpkin. So I guess git did some things and thought this was the efficient way to patch it.